src/Orleans.EventSourcing.Snapshot/LogViewAdaptor.cs

```diff
@@ -269,17 +269,6 @@ protected override async Task<int> WriteAsync()
                 }
             }
 
-            if (batchSuccessfullyWritten)
```

Comment:
Why delete it?

Reply:
BroadcastNotification is no longer supported by Orleans: https://github.com/dotnet/orleans/commit/4f436b2cf070448bc4e3d3592b0a69d870001916#diff-6c4d570757b03f93b2c4294343df1585f2a6aab49d5be2d448ad52435d1b8b23